dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/IndiciesInfo.java

```diff
@@ -0,0 +1,234 @@
+        final String timeStamp = timestampFormatter.format(new Date());
+
+        for (final IndexType indexType : indiciesType) {
+            final String indexName = String.format(
+                    INDEX_NAME_PATTERN,
+                    ClusterFactory.getClusterId(),
+                    indexType.getPrefix(),
+                    timeStamp);
+
+            this.indiciesNames.put(indexType, indexName);
+        }
+
+        return timeStamp;
+    }
+
+    public Map<IndexType, String> asMap() {
+        final Map<IndexType, String> actives = new HashMap<>();
+        for (IndexType type : IndexType.values()) {
+            final String newValue = Try.of(() -> (String) PropertyUtils
+                    .getProperty(this, type.getPropertyName())).getOrNull();
+            if (newValue != null) {
+                actives.put(type, newValue);
+            }
+
+        }
+        return actives;
+    }
+
+    @Override
+    public int hashCode() {
+        final int prime = 31;
+        int result = 1;
+        result = prime * result + ((this.getLive() == null) ? 0 : this.getLive().hashCode());
+        result = prime * result + ((this.getReindexLive() == null) ? 0 : this.getReindexLive().hashCode());
+        result = prime * result + ((this.getReindexWorking() == null) ? 0 : this.getReindexWorking().hashCode());
+        result = prime * result + ((this.getSiteSearch() == null) ? 0 : this.getSiteSearch().hashCode());
+        result = prime * result + ((this.getWorking() == null) ? 0 : this.getWorking().hashCode());
+        return result;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (this == obj)
+            return true;
+        if (obj == null)
+            return false;
+        if (getClass() != obj.getClass())
+            return false;
+        IndiciesInfo other = (IndiciesInfo) obj;
+        if (this.getLive() == null) {
+            if (other.getLive() != null)
+                return false;
+        } else if (!this.getLive().equals(other.getLive()))
+            return false;
+        if (this.getReindexLive() == null) {
+            if (other.getReindexLive() != null)
+                return false;
+        } else if (!this.getReindexLive().equals(other.getReindexLive()))
+            return false;
+        if (this.getReindexWorking() == null) {
+            if (other.getReindexWorking() != null)
+                return false;
+        } else if (!this.getReindexWorking().equals(other.getReindexWorking()))
+            return false;
+        if (this.getSiteSearch() == null) {
+            if (other.getSiteSearch() != null)
+                return false;
+        } else if (!this.getSiteSearch().equals(other.getSiteSearch()))
+            return false;
+        if (this.getWorking() == null) {
+            if (other.getWorking() != null)
+                return false;
+        } else if (!this.getWorking().equals(other.getWorking()))
+            return false;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This statement should have braces](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=3712629)